graph-refresh/overlays/stage/cronjob.yaml

```diff
@@ -4,4 +4,4 @@ apiVersion: batch/v1beta1
 metadata:
   name: graph-refresh
 spec:
-  suspend: false
+  suspend: false
```

Comment:
please remove the newline here as well,
no changes were needed in this file.

Reply:
removed :)